#436 Fix side navigation menu in Firefox at smaller screen widths

jjhampton opened the pull request:
# Description of changes
Adjusted CSS position rules for the sideNav component and the drawer component. The drawer is intended to be a reusable layout component (that can potentially be used for other things besides wrapping the sideNav's contents - anything else we want to slide out from the left for some reason.) When the drawer is visible, it needs full-screen height & width, so it gets the `position: fixed`.  The sideNav doesn't need that style.  

Seems like the `position: fixed` applied directly to the sideNav `.content` may have been interfering with the CSS transition in Firefox somehow. 

I tested this on current versions of Chrome, Firefox, Edge (Windows) and Chrome for Android - the  UI behaved as intended (no lock up or misrendering as was occurring on Firefox earlier) ... Recommend that someone who has an iPhone or Mac test it on those devices for regression testing @kylemh @Cooperbuilt 

# Issue Resolved
<!-- Keeping the format 'Fixes #123' will automatically close the issue when this PR is merged -->
Fixes #436

kylemh commented on src/scenes/home/header/sideNav/sideNav.css:
```diff
@@ -3,7 +3,6 @@
-  position: fixed;
 }
 
 .header {
```
Change .header {...} to
```
.header {
  display: inline-block;
  line-height: 0;
}
```
Set .close top to 27px and .logo top to 25px

Delete .logoWrapper styles entirely.

jjhampton commented:
@kylemh Thanks for the review - I added a few more CSS tweaks as well - used CSS transforms to vertically center the .close and .logo instead of using the px values.  

Removed the unneeded `display: flex` from the .header and instead gave it a set height, which allows us to get rid of the `margin-top` from the list below it. Should be the same height as the page header (which is visible below the transparent sideNav).
